woocommerce/woocommerce
# CYS: Ensure that the button in the Intro page redirects to the Customizer when a classic theme is enabled

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR ensures that the button in the Intro page redirects to the Customizer when a classic theme is enabled.

![image](https://github.com/user-attachments/assets/e2238beb-0069-4399-aa93-7dfd72239617)

I think that we should start to document all the possibility flows; otherwise, in the future, it will be complicated to understand this part of the project. What do you think, @albarin?




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable a classic theme like Storefront.
2. Head over to WooCommerce > Home 
3. Ensure that Customize Your Store step isn't marked.
4. Click on Customize your store.
5. Ensure that the button looks like the attached image.
6. Enable a block theme
7. Head over to WooCommerce > Home > Customize Your Store
8. Follow the flow.
9. Save.
10. Enable a classic theme like Storefront.
11. Head over to WooCommerce > Home.
12. Ensure that Customize Your Store step is marked.
13. Click on Customize your store.
14. Ensure that the button looks like the attached image.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Ensure that the button in the Intro page redirects to the Customizer when a classic theme is enabled.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
